RoaringBitmap/src/main/java/org/roaringbitmap/longlong/ImmutableLongBitmapDataProvider.java

```diff
@@ -142,6 +143,22 @@ public default LongStream reverseStream() {
    */
   public long select(long j);
 
+  /**
+   * Get the first (smallest) integer in this RoaringBitmap,
+   * that is, returns the minimum of the set.
+   * @return the first (smallest) integer
+   * @throws NoSuchElementException if empty
+   */
+  long first();
+
+  /**
+   * Get the last (largest) integer in this RoaringBitmap,
+   * that is, returns the maximum of the set.
```

Comment:
```suggestion
   * that is, return the maximum of the set.
```

Reply:
Just reproduced as in RoaringBitmap. Fixing both.